tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,3100 @@
+    "\t\t\t[1.4,  1.5, 3.3],\n",
+    "\t\t], \n",
+    "\t],\n",
+    "\tcoords=[[10, 20], [0, 1, 2], [5, 6, 7]], dims=[\"height\", \"lat\", \"lon\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6661e3fe-1f01-4fcd-a7ec-353a1a8e3a04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Within 0.1 or 0.2 of the forecast in all cases except one\n",
+    "# Can be used to find some exact matches, and some close matches\n",
+    "simple_obs = xr.DataArray(\n",
+    "    [\n",
+    "\t\t[\n",
+    "\t\t\t[0.9, 0.0,   5], \n",
+    "\t\t\t[0.7, 1.3, 2.7],\n",
+    "\t\t\t[.3,  0.4, 2.2],\n",
+    "\t\t], \n",
+    "\t\t\t[\n",
+    "\t\t\t[1.7, 1.2,  1.7], \n",
+    "\t\t\t[1.7, 2.2,  3.9],\n",
+    "\t\t\t[1.6,  1.2, 9.9],\n",
+    "\t\t], \n",
+    "\t],\n",
+    "\tcoords=[[10, 20], [0, 1, 2], [5, 6, 7]], dims=[\"height\", \"lat\", \"lon\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3b9a4ad-3a63-45ee-90f6-bf2087aa18cf",
+   "metadata": {},
+   "source": [
+    "For the purposes of this example, let's say an event occurs if the precipitation value is greater than 1.3. We therefore define a threshold based event operator with a threshold of 1.3.\n",
+    "\n",
+    "Applying this operator, means `scores` converts the non-binary data into binary data. It does this by creating binary event data - precipitation data with a value less than or equal to 1.3 is \"false\" and precipitation data with a value greater than 1.3 is \"true\".\n",
+    "\n",
+    "The following code creates a \"greater than\" operator  -  using `scores` built-in ThresholdEventOperator  -  which creates events with a threshold of \"> 1.3\". It can be repeatedly called with different event thresholds, for reasons that will be explained later.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "70a9fd8d-af7b-472a-a6ab-7355a510a6b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# An event here is defined as precipitation with a value greater than 1.3\n",
+    "\n",
+    "# The EventThresholdOperator can take a variety of operators from the python \"operator\" module, or a user-defined function\n",
+    "# The default is operator.gt, which is the same as \">\" but in functional form.\n",
+    "\n",
+    "event_operator = scores.categorical.ThresholdEventOperator(default_event_threshold=1.3)\n",
+    "\n",
+    "# https://docs.python.org/3/library/operator.html provides an overview of what standard operator types can be supplied\n",
+    "# You may wish to exeriment with the following alternatives now\n",
+    "\n",
+    "# Greater-than (not equal to) operator\n",
+    "# event_operator = scores.categorical.ThresholdEventOperator(default_event_threshold=1.3, default_op_fn=operator.gt)\n",
+    "\n",
+    "# Less-than operator\n",
+    "# event_operator = scores.categorical.ThresholdEventOperator(default_event_threshold=1.3, default_op_fn=operator.lt)"
+   ]
```

Comment:
Consider using markdown here.

Reply:
I'd prefer to leave it as-is, because people can just uncomment and run the lines of code as things stand